easybuild/easyconfigs/g/gffcompare/gffcompare-0.10.6-GCCcore-7.3.0.eb

```
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'gffcompare'
```

The name is written like this on the website and in existing easyconfigs:

```suggestion
name = 'GffCompare'
```